angularSTEP/src/app/app.module.ts

```diff
@@ -101,14 +103,14 @@ import {ViewProfilesComponent} from './view-profiles/view-profiles.component';
     MatButtonModule,
     MatCardModule,
     MatChipsModule,
-    MatDialogModule,
     MatDividerModule,
     MatExpansionModule,
     MatFormFieldModule,
-    MatIconModule,
+    MatSidenavModule,
     MatInputModule,
+    MatDialogModule,
+    MatIconModule,
```

Comment:
Does `MatListModule` not need to be declared here or do I just not see it?

Reply:
Oh yes, thank you